### PR TITLE
Doc update readme

### PR DESCRIPTION
**Pull request by Inksprout**
- update readme with windows instructions and other clean up
- remove src folder in file structure as it's not needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,20 +11,21 @@ The project uses a Makefile to simulate a very simple build pipeline with two st
 **Software**:
 
 * Tools listed at: https://docs.pactflow.io/docs/workshops/ci-cd/set-up-ci/prerequisites/
-* A pactflow.io account with an valid [API token](https://docs.pactflow.io/docs/getting-started/#configuring-your-api-token)
-* .NET core 3.1 SDK installed, this is the version of .NET core used for this example
+* A pactflow.io account with an valid [API token](https://docs.pactflow.io/#configuring-your-api-token)
+* [.NET core 3.1 SDK installed](https://dotnet.microsoft.com/en-us/download/dotnet/3.1)d, this is the version of .NET core used for this example
```
Comment (YOU54F) [inline README.md]: trailing d after hyperlink 👍🏾

File: README.md [hunk 2]
```diff
@@ -11,20 +11,21 @@ The project uses a Makefile to simulate a very simple build pipeline with two st
...
 * `PACT_BROKER_TOKEN`: a valid [API token](https://docs.pactflow.io/#configuring-your-api-token) for Pactflow
 * `PACT_BROKER_BASE_URL`: a fully qualified domain name with protocol to your pact broker e.g. https://testdemo.pactflow.io
+* `GIT_COMMIT`: version of the consumer app, used to publish the pact
 
 ## Usage
 
 ### Publish example to your Pactflow
 
-To publish this example to your own Pactflow account and view it in the UI follow these steps.
+To publish this example to your own Pactflow account and view it in the UI follow these steps, note these instructions are for UNIX based OS.
```
Comment (YOU54F) [inline README.md]: UNIX based OS's (Linux / Mac / Windows via WSL only) - See ### Using Example on Windows

👍🏾

File: README.md [hunk 3]
```diff
@@ -37,10 +38,69 @@ To publish this example to your own Pactflow account and view it in the UI follo
...
-### Using the fake CI
+   
+5.  Using the fake CI
 * `make test` - run the tests locally
-* `make fake_ci` - run the CI process locally
+* `make fake_ci` - run the CI process locally, this will publish the Swagger document and Shchemathesis report to Pactflow.
+ 
+### Using Example on Windows.
+You can still try this example locally on Windows using powershell and running commands manually. These will be the same commands that are used in the makefile with a few manual tweaks.
+
```
Comment (YOU54F) [inline README.md:48]: To run this example on Windows, we will use Powershell and run the commands defined in the Makefile, manually. 

We will take you through the steps, but you can see the commands in the `./Makefile`

Some notes

- Most of them use the pact-cli docker image
- can be ran locally from Powershell by changing the unix paths to windows paths
- Replace the environment variable references. Any variable referenced as `${VARIABLE}` can be changed to `$env:VARIABLE` for use Powershell.

File: README.md [hunk 4]
```diff
@@ -37,10 +38,69 @@ To publish this example to your own Pactflow account and view it in the UI follo
...
+            "content": "<report.txt encoded content here>",
+            "contentType": "text/plain"	,
+            "verifier": "Schematheis"
+        }
+
+    }
+    ```
+    Send the request and when you have a 200 response the provider contract will be visible in your Pactflow account.
+
+ 4. Check can-i-deploy to see if your provider contract is compatible with current consumers. In Powershell set the environment variable $env:PACTICIPANT to the name of the provider in the url for step 3. Set the environment variable $env:GIT_COMMIT to the same thing as in step 3. This tells pactflow which provider contract and version to perform the can-i-deploy check on. Run the can-i-deploy check using the pact-cli docker container:
```
Comment (YOU54F) [inline README.md]: Check `can-i-deploy` to see if your provider contract is compatible with current consumers. 

In Powershell 
- set the environment variable `$env:PACTICIPANT` to the name of the provider in the url for step 3. 
- set the environment variable `$env:GIT_COMMIT` to the same as in step 3. 

This tells pactflow which provider contract and version to perform the `can-i-deploy` check on. 

Run the `can-i-deploy` check using the pact-cli docker container:

File: README.md [hunk 5]
```diff
@@ -37,10 +38,69 @@ To publish this example to your own Pactflow account and view it in the UI follo
...
+
+    ```
+    docker run --rm -v <path_to_project_root>:<path_to_project_root> -e PACT_BROKER_BASE_URL -e PACT_BROKER_TOKEN pactfoundation/pact-cli broker can-i-deploy --pacticipant $env:PACTICIPANT --version $env:GIT_COMMIT --to-environment production
+    ```
+
+ 5.  Have a look at what other commands are available in the Makefile. Most of them  use the pact -cli docker image and can be ran locally from Powershell by changing the windows paths to UNIX and replacing the environment variable references. Any variable referenced as `${VARIABLE}` can be changed to `$env:VARIABLE` to reference environment variables in Powershell.
```
Comment (YOU54F) [inline README.md]: moved this to the top of the steps and reworded slightly

**Review by YOU54F — APPROVED** [review]
Looks awesome @Inksprout ! 

I made a couple of suggestions for readability and suggested some ordering but great work even as it is, I know it wasn't easy on my side dusting off the windows laptop, having it finish a load of updates before I could actually start playing around with these examples

File: example-bi-directional-provider-dotnet/scripts/verify_swagger.sh [hunk 6]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
```
Comment (mefellows) [inline example-bi-directional-provider-dotnet/scripts/verify_swagger.sh:1]: It looks like you've removed execute permissions from this script - accident?

File: example-bi-directional-provider-dotnet/scripts/publish.sh [hunk 7]
```diff
@@ -5,10 +5,9 @@ if [ "${1}" != true ]; then
 	SUCCESS=false
 fi
```
Comment (mefellows) [inline example-bi-directional-provider-dotnet/scripts/publish.sh:7]: It looks like you've removed execute permissions from this script - accident?

**Review by mefellows — APPROVED** [review]
Looks good - `make fake_ci` works locally for me now 🤷 

The shell scripts have lost their execute permissions, so once you fix that (and address Yousaf's 👌  comments) we're good!

**Pull request merged**